sklearn/preprocessing/data.py

```diff
@@ -229,11 +233,17 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
 
     scale_ : ndarray, shape (n_features,)
         Per feature relative scaling of the data.
+
+
+
     """
 
-    def __init__(self, feature_range=(0, 1), copy=True):
+    def __init__(self, feature_range=(0, 1), copy=True, selected="all"):
         self.feature_range = feature_range
         self.copy = copy
+        self.selected = selected
+        self.scaled_features_ = None if selected == "all" else selected
```

Comment:
> Should I still move it to fit?

You can create a '_check_params' method, called both at the end of init
and the beginning of fit.